jmix-quartz/quartz/src/main/java/io/jmix/quartz/service/QuartzService.java

```diff
@@ -115,12 +116,17 @@ from the startTime to the current moment after trigger is recreated (all trigger
                             isActive = true;
                         }
                     }
-
                     jobModel.setTriggers(triggerModels);
+                    isRunning = scheduler.getCurrentlyExecutingJobs().stream()
```

Comment:
Maybe we should get it once, transform into more suitable form (map\set) and then check job keys.

Reply:
moved to map builded before main loop